### PR TITLE
fix: observability

### PR DESCRIPTION
**Pull request by nferch**
* create Node helper
* move/align common telemetry tags to Observability
* rejigger observability tag handling

### CODE REVIEW

File: packages/sdk/observability/src/helpers/common.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//  - Identifier can be synced via HALO to allow for correlation of events bewteen devices.
+//  - Identifier should also be stored outside of HALO such that it is available immediately on startup.
+export const getTelemetryIdentifier = (client: Client) => {
+  if (!client?.initialized) {
+    return undefined;
+  }
+
+  const identity = client.halo.identity.get();
+  if (identity) {
+    return identity.identityKey.toHex();
```
Comment (wittjosiah) [inline packages/sdk/observability/src/helpers/common.ts:19]:
```suggestion
    return identity.identityKey.truncate();
```
to align with other places we're sending this?

File: packages/sdk/observability/src/helpers/index.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@
 // Copyright 2022 DXOS.org
 //
 
-export * from './app-observability';
+export * from './browser-observability';
+export * from './node-observability';
+export * from './common';
 export * from './map-spaces';
 export * from './setup-telemetry-listeners';
```
Comment (wittjosiah) [inline packages/sdk/observability/src/helpers/index.ts:9]: sort? 🙂

File: packages/sdk/observability/src/helpers/node-observability.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+  if (existsSync(configDir)) {
+    if (!statSync(configDir).isDirectory()) {
+      throw new Error(`Config directory ${configDir} exists but is not a directory`);
+    }
+  } else {
+    await mkdir(configDir, { recursive: true });
+  }
+
+  // named 'telemetry.yml' for backwards compatibility
```
Comment (wittjosiah) [inline packages/sdk/observability/src/helpers/node-observability.ts:45]: fwiw probably will hurt less to change this sooner rather than later

**Review by wittjosiah — APPROVED** [review]

File: packages/sdk/observability/src/helpers/common.ts (hunk 1 above)
Comment (nferch) [reply packages/sdk/observability/src/helpers/common.ts:19]: whoops, must have missed this in the merge

File: packages/sdk/observability/src/helpers/node-observability.ts (hunk 3 above)
Comment (nferch) [reply packages/sdk/observability/src/helpers/node-observability.ts:45]: will change the localforage key then too?

**Pull request merged**